src/components/Jumbotron/style.scss

```diff
@@ -0,0 +1,51 @@
+.jumbo_background {
+  background-image: url("jumbotron_images/background.jpeg");
+  background-size: cover;
+  background-position: center;
+  height: 250px;
+  //display: grid;
+  //place-items: center;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.jumbo_title_box {
+  background: rgba(255, 255, 255, 0.7);
+  max-width: 90%;
+  padding: 0px 15px 15px 15px;
+}
+
+.jumbo_title {
+  color: #17a2b8;
+  font-size: 40px;
+  font-weight: 500;
+  margin: 0;
+  text-align: center;
+}
+
+.jumbo_subtitle {
+  font-size: 16px;
+  margin: 0;
+  text-align: center;
+}
```

Comment:
Because these elements are all in the jumbotron, would be worth nesting them in the .jumbotron_background selector